### ISSUE TITLE
divisions within domains

### ISSUE THREAD

**Issue by bansp** [clarin-eric/standards#51]
This is an enhancement of the idea in #42 .

These are the alphabetically ordered domains:
```
Audiovisual Annotation
Audiovisual Source Language Data
Catalogue Metadata
Contextual Data
Contextual Information
Documentation
Geodata
Image Annotation
Image Source Language Data
Language Description
Lexical Resource
Metadata
Other
Statistical Data
Text Annotation
Textual Source Language Data
Tool support
```

Alphabetical sorting was the best that we had at our disposal, but it definitely could be improved on, by introducing subdomains that might also be helpful for maintenance, aaand possibly also for users.
Technically, I would add an attribute such as "orderKey" or some such. The sort would then take first the attribute into consideration (alphabetically, why not), and then, secondary sort would be alphabetic within the key-ed group.

A very rough suggestion follows, and I'm not even sure I like it, so please feel welcome to suggest alternatives, and I hope to re-edit this:

**Annotation**:
Audiovisual Annotation
Image Annotation
Text Annotation

**Databases**:
Geodata
Language Description
Lexical Resource
Statistical Data

**Data/Resource Description**:
Catalogue Metadata
Contextual Information
Documentation
Metadata

**Source Data**:
Audiovisual Source Language Data
Contextual Data
Image Source Language Data
Textual Source Language Data

**Uncategorized** 
Other
Tool Support

**Comment by bansp** [comment]
"Tool support" could have a top-level status, given that it's fairly non-uniform.
Not sure about how to classify "Contextual data".

**Comment by hannahedeland** [comment]
Looks good to me, I would expect "Contextual Information" below "Metadata" though, and yes, maybe it could be called "Data/Resource Description" or something less specific. "Contextual Data" is fine below "Source Data" I think.

**Comment by bansp** [comment]
Putting "Documentation" under "Metadata" may be a punishable offence in some circles, so probably it's not staying. I am now wondering about two potential moves, and each of them involves a change in a name:

- either turn "Documentation" into, say, "Resource Documentation", and then take "Documentation" as the name of the superset containing {Catalogue Metadata, Resource Documentation, Metadata}
- or turn "Metadata" into... "Formal Metadata" (hmm), and, despite the inevitable punishment, have "Documentation" as the name of the superset containing {Catalogue Metadata, Resource Documentation, Formal Metadata}
(ouch, this is posted without taking Hanna's comment above into consideration, so let's see)

**Comment by bansp** [comment]
> Looks good to me, I would expect "Contextual Information" below "Metadata" though, and yes, maybe it could be called "Data/Resource Description" or something less specific. "Contextual Data" is fine below "Source Data" I think.

@hannahedeland , do you mean turning "Contextual Information" into "Data/Resource Description", or rather using "Data/Resource Description" as a header for the current super-category Metadata, and we would probably converge there? Especially if you agreed to add Documentation there? ;-)

**Comment by bansp** [comment]
Took a stab and did what I thought you suggested (or what I hoped you suggested ;-)).

**Comment by bansp** [comment]
Now, Contextual Data mutters that it's not entirely comfortable under "Source Data", because part of its definition is "but not part of the source language data", but this may be a matter of an extra pillow, rather than restructuring. The devil may sit in the "language" bit. "Source Data" is not "Source _Language_ Data", after all.

**Comment by hannahedeland** [comment]
Yes that's what i meant :) but I'm not sure about the top-level name, maybe someone wants something else.

And for Contextual Data, yes, I'd say it's not language data, but source data is fine I think.

**Comment by bansp** [comment]
Now, about technicalities. An `@orderKey` attribute seems the most flexible way out, rather than introducing any 'physical' structure in the list of domains. I would see the algorithm as 

- grouping by group-by, 
- using the current-key for the header
- enumerating the group in an alphabetical order

And using the same logic both for the side menu listing domains and in the script that pre-fills domains in the import (#42). Maybe the script could even print a comment in the XML file with the group-by key (not sure about that latter point at all -- thinking of maintenance).

**Comment by bansp** [comment]
Let me also remind us at this point, that we have kept one group for the future:

[ Query Engine Input/Tool+Service Format ]
Formats that exist only for distribution via query platforms, internal tool formats etc.

**Comment by bansp** [comment]
> I'm not sure about the top-level name, maybe someone wants something else.

@hannahedeland -- if we keep that as an attribute, it will be an easy change, should someone come up with a more fitting name.

**Comment by bansp** [comment]
Hah, so I am adding this to the upcoming milestone, because it doesn't seem like much work. I would prefer the earlier milestone, recommendations, because it's so very user-oriented and may affect the reception and uptake of the system.

**Comment by bansp** [comment]
Partially handled by commit https://github.com/clarin-eric/standards/commit/bfb1b426c60ce1ff4ea6431be663016d5d340bbf

**Comment by bansp** [comment]
Done in https://github.com/clarin-eric/standards/commit/a9fb4d35e1aadb71251ec024bb425fbfe47538e4 although Eliza will probably want to correct the styling to make it look nicer.

**Comment by bansp** [comment]
Closed with the merger of PR #53 .
